src/PowerShell/Help/Microsoft.WinGet.Client/Remove-WinGetSource.md

```diff
@@ -0,0 +1,75 @@
+﻿---
+external help file: Microsoft.WinGet.Client.Cmdlets.dll-Help.xml
+Module Name: Microsoft.WinGet.Client
+ms.date: 08/01/2024
+online version:
+schema: 2.0.0
+title: Remove-WinGetSource
+---
+
+# Remove-WinGetSource
+
+## SYNOPSIS
+Removes a configured source.
+
+## SYNTAX
+
+```
+Remove-WinGetSource -Name <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This command removes a configured WinGet source. By default, there are two sources registered:
+`msstore` and `winget`. You can add more sources using `Add-WinGetSource`.
```

Comment:
This has to be run as admin, right? Maybe we should clarify it like for other commands.

Reply:
Added description that add/remove source commands must be run as admin.